_pages/oidc.md

```diff
@@ -195,6 +195,7 @@ In an **unsuccessful authorization**, the URI will contain the parameters `error
   - `invalid_request` — The authorization request was invalid. See the `error_description` parameter for more details.
 - **error_description** — A description of the error.
 - **state** — The `state` value originally provided by the client.
+- **id_token** - (See id_token in [token response](#token-response) for more detail)
```

Comment:
This is wrong. The redirect URI from the authorization point does not contain the ID token

Reply:
Here's the code for the success redirect URI:
https://github.com/18F/identity-idp/blob/master/app/forms/openid_connect_authorize_form.rb#L87-L92

and the error redirect URI:
https://github.com/18F/identity-idp/blob/master/app/forms/openid_connect_authorize_form.rb#L194-L203

we never give back the `id_token` in the redirect URI because redirects are easily intercepted and we don't want to give away PII like that